src/common/styles/Table.tsx

```diff
@@ -2,22 +2,22 @@ import styled from "@emotion/styled";
 import tw from "twin.macro";
 
 export const StyledTable = styled.div`
-  ${tw`w-full border border-solid border-grey-300 overflow-x-auto scrolling-touch`}
+  ${tw`w-full border border-solid border-gray-300 overflow-x-auto`}
```

Comment:
let's add them back as custom utilities. although not a web standard, it's the quickest fuss free way to ensure inertia scrolling in touch devices